scripts/copy_output.lua

```diff
@@ -64,5 +64,5 @@ if lib.opkg and (env.GLUON_DEVICES or '') == '' then
 	lib.exec {'rm', '-f', dest_dir('\0')..'/\0'}
 	lib.exec({'rmdir', '-p', dest_dir('\0')}, true, '2>/dev/null')
 	mkdir(dest_dir(package_prefix))
-	lib.exec {'cp', 'openwrt/bin/targets/'..bindir..'/packages/\0', dest_dir(package_prefix)}
+	lib.exec {'mv', 'openwrt/bin/targets/'..bindir..'/packages/\0', dest_dir(package_prefix)}
```

Comment:
Removing this directory from the OpenWrt dir should be avoided, it will significantly increase incremental build time with `GLUON_DEVICES` during development (see `clean_output.lua`).